packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-logs/job-logs.controller.js

```diff
@@ -37,21 +39,18 @@ export default class {
   }
 
   downloadLogs() {
-    const re = /https:\/\/storage\.[a-z0-9]+\.cloud.ovh.net\/v1\/AUTH_[a-z0-9]+\/(.*)\/(.*)/;
-    const logsUrl = this.logger.logs.logsAddress;
-    if (logsUrl) {
-      const matches = logsUrl.match(re);
-      this.logger.downloadObject(
-        this.projectId,
-        this.job.region,
-        matches[1],
-        matches[2],
-      );
-      this.isDownloadButtonDisabled = true;
-      this.$timeout(() => {
-        this.isDownloadButtonDisabled = false;
-        this.$scope.$apply();
-      }, 3000);
-    }
+    // Get all the object storages available
+    this.PciStoragesContainersService.getAll(this.projectId, false).then(
+      (containers) => {
+        // The logs are stored inside the 'odp-logs' storage so we're looking for this one
+        const container = containers.find(
+          (storage) => storage.name === 'odp-logs',
+        );
+        // If we find it, we get the id and redirect to the object storage page
+        if (container) {
+          this.redirectToObjectStorage(this.projectId, container.id);
```

Comment:
As the `$state.go` returns a promise
```suggestion
          return this.redirectToObjectStorage(this.projectId, container.id);
```

Reply:
Done